apps/src/templates/teacherHomepage/SetUpMessage.jsx

```diff
@@ -63,7 +68,12 @@ const SetUpMessage = React.createClass({
           <div style={styles.description}>
             {i18n.createNewClassroom()}
           </div>
-          <ProgressButton href="wherever we go to create sections" color="gray" text={i18n.createSection()} style={styles.button}/>
+          <ProgressButton
+            href="https://code.org/teacher-dashboard#/sections"
```

Comment:
would be good if we didn't hardcode to production environment. In some places we accomplish that by using `window.dashboard.CODE_ORG_URL`. I believe this is pegasus, and we don't have that available to us? If so, we should try to come up with something equivalent.